pixeltable/catalog/table_version.py

```diff
@@ -557,7 +557,7 @@ def drop_column(self, name: str) -> None:
             )
         # See if this column has a dependent store. We need to look through all stores in all
         # (transitive) views of this table.
-        transitive_views = Catalog.get().tbls[self.id].get_views(recursive=True)
+        transitive_views = Catalog.get().tbls[self.id]._get_views(recursive=True)
```

Comment:
It doesn't really make sense for TableVersion to be interacting with Table, the latter is only meant as the user-facing interface to tables. get_views() needs to be in TableVersion, not Table.